packages/lesswrong/lib/collections/posts/views.ts

```diff
@@ -413,18 +445,6 @@ ensureIndex(Posts,
     name: "posts.afSticky_score",
   }
 );
-ensureIndex(Posts,
-  augmentForDefaultView({ metaSticky:-1, score:-1 }),
-  {
-    name: "posts.metaSticky_score",
-  }
-);
-ensureIndex(Posts,
-  augmentForDefaultView({ userId: 1, hideAuthor: 1, ...stickiesIndexPrefix, score:-1 }),
-  {
-    name: "posts.userId_stickies_score",
```

Comment:
This was added in commit 7fb9b1e3f1ccc775e4ed326b0f41fcc2b3fd7750, which was labeled "Restore view that was misidentified as unused". So, some prior this is actually used somehow, but I don't see how.